Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -528,6 +536,34 @@ class OverviewViewController: UIViewController {
         }
     }
     
+    @IBAction func showHideTapped(_ sender: Any) {
+        self.setShowHideBalance(value: !self.getShowHideBalance())
+        self.showHideButton.forEach { button in
+            button.setImage(UIImage(named: self.getShowHideBalance() ? "ic_conceal" : "ic_reveal"), for: .normal)
+            if button.tag == 3 {
+                button.isHidden = self.pageSubtitleLabel.isHidden
+            }
+            if button.tag == 0 {
+                button.isHidden = !self.pageSubtitleLabel.isHidden
+            }
+        }
+        self.updateBalance()
+    }
+    
+    func getShowHideBalance() -> Bool {
+        return WalletLoader.shared.multiWallet.readBoolConfigValue(forKey: GlobalConstants.Strings.SHOW_HIDE_BALANCE, defaultValue: true)
```

Comment:
```suggestion
        return WalletLoader.shared.multiWallet.readBoolConfigValue(forKey: GlobalConstants.Strings.SHOW_HIDE_BALANCE, defaultValue: false)
```
default should be false